fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/BaseDataServletComponentSpec.groovy

```diff
@@ -206,23 +203,22 @@ abstract class BaseDataServletComponentSpec extends Specification {
      */
     Response makeAbstractRequest(Closure queryParams=this.&getQueryParams) {
         // Set target of call
-        def httpCall = jtb.getHarness().target(getTarget())
+        def httpCall = jtb.harness.target(target)
```

Comment:
Your example doesn't quite capture my concern. Here's a better one:

```groovy
class User {
    public final String name                 
    User(String name) { this.name = name}
    String getName() { "Name: $name" }
    void printName() {
         println(name)
     }
}
class OtherUser {
    String getName() { "Name: name" }
    void printName() { println(name) }
}

def user = new User('Bob')
assert user.name == 'Name: Bob'   // Goes through the getter
assert user.@name == 'Bob' // Gets the field directly
user.printName()
user.printName()
new OtherUser().printName()
```
Output:
```
Bob
Name: name
```
In other  words, inside of a class, the field takes precedence over the getter. Which means adding a private field may have very unexpected changes to behavior. 

So I would personally encourage that we should only use the shortcut outside of a class. When inside a class, we should always explicitly call the method.